frontend/src/Pages/Admin.js

```diff
@@ -44,7 +44,7 @@ const Admin = () => {
               <AdminDeleteProduct productID={products.productID} />
             </div>
             <img className="pimage rounded-2 mb-2" src={products.image}></img>
-            <h6> £{products.price}</h6>
+            <h6> £{products.price.toFixed(2)}</h6>
```

Comment:
Screenreaders use `<h>` tags to aid navigation by jumping between sections on the page, so it's best to use them for actual section headings and stick with other elements like `<p>` or `<span>` tags for regular text.

Reply:
Applied this change 👍